src/operator/controllers/intents_reconcilers/port_egress_network_policy/port_egress_network_policy.go

```diff
@@ -96,90 +96,87 @@ func (r *PortEgressNetworkPolicyReconciler) Reconcile(ctx context.Context, req c
 			r.RecordWarningEventf(intents, consts.ReasonNamespaceNotAllowed, "ClientIntents are in namespace %s but namespace is not allowed by configuration", intents.Namespace)
 			continue
 		}
-		createdPolicies, err := r.handleNetworkPolicyCreation(ctx, intents, intent, req.Namespace)
+		policy, err := r.handleNetworkPolicyCreation(ctx, intents, intent, intents.Namespace)
 		if err != nil {
 			r.RecordWarningEventf(intents, consts.ReasonCreatingEgressNetworkPoliciesFailed, "could not create network policies: %s", err.Error())
-			return ctrl.Result{}, errors.Wrap(err)
+			return nil, errors.Wrap(err)
 		}
-		if createdPolicies {
-			createdNetpols += 1
+		if policy != nil {
```

Comment:
When is policy nil?
This should never happen - if err == nil then the other param should never be nil.